### PR TITLE
[WIP] - Add sdk for all endpoints

### PR DESCRIPTION
**Pull request by sujit-baniya**
https://app.asana.com/0/802457876449163/1201463708923484

### CODE REVIEW

File: contact.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+require ('./vendor/autoload.php');
+
+$credentials = null;
+require_once 'auth.php';
+
+echo 'Contact';
+$contact = new \Intergo\SmsTo\Module\Contact\Contact($credentials);
+
+// @TODO - Remove this line before the package go live
```
Comment (panicoschr) [inline contact.php]: Shouldn't these comment be removed or you plan to use them later?

File: number-lookup.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+$credentials = null;
+require_once 'auth.php';
+
+echo 'Number Lookup';
+$hlr = new \Intergo\SmsTo\Module\NumberLookup\NumberLookup($credentials);
+
+// @TODO - Remove this line before the package go live
+$hlr->setBaseUrl('https://smsto.local.intergo.co');
+
```
Comment (panicoschr) [inline number-lookup.php]: here you have var_dump, shouldn't these be removed?

File: shortlink.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+echo 'Shortlink';
+$shortlink = new \Intergo\SmsTo\Module\Shortlink\Shortlink($credentials);
+
+// @TODO - Remove this line before the package go live
+$shortlink->setBaseUrl('https://smsto.local.intergo.co');
+
+// var_dump($shortlink->create('Sujit', 'https://sujitbaniya.com'));
+// var_dump($shortlink->all());
+var_dump($shortlink->getByID(69));
```
Comment (panicoschr) [inline shortlink.php]: same here?

File: sms.php [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+echo 'SMS Messaging';
+$sms = new Sms\Sms($credentials);
+
+// @TODO - Remove this line before the package go live
+$sms->setBaseUrl('https://msms.local.intergo.co');
+
+$message = new Message\SingleMessage();
+$message->setTo('+9779856034616');
+$message->setMessage('This is test');
+var_dump($sms->estimate($message));
```
Comment (panicoschr) [inline sms.php]: var_dump here?

File: sms.php [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+$message = new Message\CampaignMessage();
+$message->setTo(['+9779856034616', '+9779805832689']);
+$message->setMessage('This is test');
+var_dump($sms->estimate($message));
```
Comment (panicoschr) [inline sms.php]: same

File: sms.php [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    [
+        "to" => "+9779856034616",
+        "message" => "This is test"
+    ],
+    [
+        "to" => "+9779805832689",
+        "message" => "This is another test"
+    ],
+]);
+var_dump($sms->estimate($message));
```
Comment (panicoschr) [inline sms.php]: same

File: sms.php [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+$message = new Message\ListMessage();
+$message->setListId(122);
+$message->setMessage('This is test');
+var_dump($sms->estimate($message));
```
Comment (panicoschr) [inline sms.php]: same for all rest

File: team.php [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+require ('./vendor/autoload.php');
+
+$credentials = null;
+require_once 'auth.php';
+
+echo 'Team';
+$team = new \Intergo\SmsTo\Module\Team\Team($credentials);
+
+// @TODO - Remove this line before the package go live
```
Comment (panicoschr) [inline team.php]: var_dump here?

**Review by panicoschr — APPROVED** [review]
PR Review ok.

**Review by poupouxios — APPROVED** [review]

**Pull request merged**